### PR TITLE
Update Abstract, FCU

### PR DESCRIPTION
**Pull request by Karthi8883**

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 1]
```diff
@@ -587,6 +587,22 @@ DFSS:
   - discharge_fan_run_command
   - discharge_fan_run_status
 
+DF2SS:
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:590]: You use supply and discharge in the same abstract. Fix.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 2]
```diff
@@ -587,6 +587,22 @@ DFSS:
...
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
+  - supply_air_damper_status_1
+  - return_air_damper_status_1
+  - supply_air_damper_status_2
+  - return_air_damper_status_2
+  uses:
+  - discharge_fan_run_command_1
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:601]: These should be put on their own in a type called DFSS2X, to conform with the existing convention.

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml [hunk 3]
```diff
@@ -587,6 +587,22 @@ DFSS:
   - discharge_fan_run_command
   - discharge_fan_run_status
 
+DF2SS:
+  description: "Two discharge fan run command and status (start/stop). With dampers"
+  is_abstract: true
+  implements:
+  - OPERATIONAL
+  opt_uses:
+  - supply_air_damper_status_1
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:596]: these can be two separate types, following the convention of `RAIDC3X`; call them `DAIDC2X` and `RAIDC2X` and add isolation to the fields if they are for isolation.

**Review by tasodorff — CHANGES_REQUESTED** [review]

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 3 above)
Comment (Karthi8883) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:596]: Modified

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 2 above)
Comment (Karthi8883) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:601]: Modified

File: ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml (hunk 1 above)
Comment (Karthi8883) [reply ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml:590]: Modified

**Review by tasodorff — APPROVED** [review]

**Comment by mschulze17** [comment]
@cstirdivant yours to merge

**Pull request merged**